app/workers/email_notification_worker.rb

```diff
@@ -0,0 +1,9 @@
+require 'sidekiq-scheduler'
+
+class EmailNotificationWorker
+  include Sidekiq::Worker
+  def perform
+    NotificationMailer.notification_email.deliver
```

Comment:
Sidekiq ensures jobs are delivered **at least once**, so you might want to make sure you're not overdelivering the same notification.